apps/onlyoffice.sh

```diff
@@ -23,6 +23,51 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+## Onlyoffice for NC 18.0.1 onwwards
+# Check for the NC-version
+CURRENTNCVERSION="$(occ_command config:system:get version | sed 's/^\(.*\..*\..*\)\..*/\1/')"
+CURRENTNCVERSION="${CURRENTNCVERSION//./}"
+if [ "$CURRENTNCVERSION" -lt "9999" ]
+then
+    CURRENTNCVERSION="$(sed 's/./&0/3' <<<"$CURRENTNCVERSION")"
+fi
+
+# Check if onlyoffice is already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice or Collabora is already installed..."
+if [ "$CURRENTNCVERSION" -ge 18001 ] && ! does_this_docker_exist 'onlyoffice/documentserver' || ! does_this_docker_exist 'collabora/code'
+then
+    install_if_not jq
+    if occ_command_no_check app:list --output=json | jq -e '.enabled | .documentserver_community' > /dev/null
```

Comment:
If you implement a new standard somewhere, please check the rest of the scripts as well and implement the same standard everywhere.

Reply:
okay, I think we already have a few places. so I will do that later on.

Reply:
I will do that in another PR, when this two are merged.
https://github.com/nextcloud/vm/issues/1082